cmd/operator-sdk/generate/csv.go

```diff
@@ -238,6 +243,13 @@ func (c csvCmd) run() error {
 	if c.operatorName == "" {
 		c.operatorName = filepath.Base(projutil.MustGetwd())
 	}
+
+	// Provide interactive propmpts to let the user provide inputs
+	// for CSV fields.
+	if c.enableCSVPrompt == true {
+		c.interactivePrompt()
```

Comment:
Semi-blocking: `generate csv` should only ask for interactive input if it cannot find a base CSV to read metadata from. This single source of metadata will not explicitly exist until at least part of #2860 is merged. In the mean time we could check if `deploy/olm-catalog/<operator-name>/manifests/<operator-name>.clusterserviceversion.yaml` exists, and if not begin an interactive prompt.

Reply:
I was wondering if there can be a case, where even if there is a base CSV, the user still wants to modify the metadata (especially UI metadata like adding/changing the keywords, description etc).Wouldn't these interactive prompts make it easier than making the user to manually go and modify it. And if there seems to be a case, where we have a baseCSV from where all the metadata can be taken from, and the user does not want to update it, they can use `--interactive` flag to skip it. But yes, I think this would change once we have `--generate bundle` merged. Thoughts on this?

Reply:
Perhaps setting `--interactive=false` by default and starting an interactive prompt in case a base CSV does not exist would be better. Then an interactive prompt could be requested explicitly by setting `--interactive`. I suggest this flow because ideally an interactive prompt should only be needed once when initializing OLM manifests, but having the option to enable it whenever should be available.